src/wordpiece_tokenizer.cpp

```diff
@@ -2,16 +2,14 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-#ifdef ENABLE_FAST_TOKENIZERS
```

Comment:
Move `#include "wordpiece_tokenizer.hpp"` statement in `tokenizer.hpp`

Reply:
done